db/migrate/20190717074405_add_profile_fields_to_user.rb

```diff
@@ -0,0 +1,10 @@
+class AddProfileFieldsToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :location, :string
+    add_column :users, :my_plants, :string
```

Comment:
I think adding this will be a separate feature, you will probably have another model for plants

Reply:
removed my_plants

Reply:
Let's keep it for now